pkg/ebpf/c/bpf_metadata.h

```diff
@@ -0,0 +1,17 @@
+/**
+ * This header file is included in kconfig.h and ktypes.h, which should make
```

Comment:
If this is the case, why is this being added to all the .c files in this PR? Is there a way to check if this is not included in a .c file?

Reply:
This was the comment from the previous PR, but we changed the approach and decided to include it in .c files directly. I updated the comment to reflect that. The invoke tasks check that the metadata is included after al .o files are built.